### PR TITLE
Issue 32

### PR DESCRIPTION
**Pull request by jonlouie**
*Issue #, if available:* https://github.com/aws/cta/issues/32

*Description of changes:*
* Add unit tests for porting rules added here: 
* Add property initialization to prevent NPE's

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/CTA.Rules.Models/ProjectActions.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ public ProjectActions()
             FileActions = new BlockingCollection<FileActions>();
             PackageActions = new BlockingCollection<PackageAction>();
             ProjectReferenceActions = new BlockingCollection<string>();
+            ProjectLevelActions = new List<ProjectLevelAction>();
```
Comment (marknfawaz) [inline src/CTA.Rules.Models/ProjectActions.cs:17]: Do we need this?

File: tst/CTA.Rules.Test/PortCoreTests.cs [hunk 2]
```diff
@@ -276,6 +275,24 @@ private void TestMvcMusicStore(string version)
             Assert.AreEqual(mvcProjectActions.Count, 4);
         }
 
+        [Test]
+        public void TestAntlrSampleSolution()
+        {
+            var version = "netcoreapp3.1";
```
Comment (Eruanion) [inline tst/CTA.Rules.Test/PortCoreTests.cs:281]: Probably want to follow the example of the other tests and pull out the version to a separate function so we can inject the version number ourselves and run .net 5 if need be.

File: src/CTA.Rules.Models/ProjectActions.cs (hunk 1 above)
Comment (Eruanion) [inline src/CTA.Rules.Models/ProjectActions.cs:17]: Does this get used anywhere for now?

Comment (jonlouie) [reply src/CTA.Rules.Models/ProjectActions.cs:17]: I was getting null reference errors for it. Not sure about other collections, but List properties aren't initialized or given a default value unless specified.

Comment (jonlouie) [reply src/CTA.Rules.Models/ProjectActions.cs:17]: Yes, it is used elsewhere in the class but was not initialized.

File: tst/CTA.Rules.Test/PortCoreTests.cs (hunk 2 above)
Comment (jonlouie) [reply tst/CTA.Rules.Test/PortCoreTests.cs:281]: Good point. I will update this.

**Review by marknfawaz — APPROVED** [review]

**Pull request merged**